### PR TITLE
stub cellHttp

### PR DESCRIPTION
**Pull request by Megamouse**
FpwV

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellHttp.h [hunk 1]
```diff
@@ -1087,3 +1087,128 @@ enum CellHttpErrorNet
...
+	CELL_HTTP_STATUS_CODE_Conflict                        = 409,
+	CELL_HTTP_STATUS_CODE_Gone                            = 410,
+	CELL_HTTP_STATUS_CODE_Length_Required                 = 411,
+	CELL_HTTP_STATUS_CODE_Precondition_Failed             = 412,
+	CELL_HTTP_STATUS_CODE_Request_Entity_Too_Large        = 413,
+	CELL_HTTP_STATUS_CODE_RequestURI_Too_Long             = 414,
+	CELL_HTTP_STATUS_CODE_Unsupported_Media_Type          = 415,
+	CELL_HTTP_STATUS_CODE_Requested_Range_Not_Satisfiable = 416,
+	CELL_HTTP_STATUS_CODE_Expectation_Failed              = 417,
+	CELL_HTTP_STATUS_CODE_Unprocessable_Entity            = 422,
```
Comment (Anuskuss) [inline rpcs3/Emu/Cell/Modules/cellHttp.h:1148]: Where [teapot](https://developer.mozilla.org/de/docs/Web/HTTP/Status/418) :thinking:

**Comment by cosmo2008** [comment]
I have just updated the new version of psc3 and it completely broke the audio on p5
It sounds distorted, I thought maybe it was a mod but no. Even after uninstalling and reseting all the settings it still wont work

**Comment by segbed** [comment]
> I have just updated the new version of psc3 and it completely broke the audio on p5 It sounds distorted, I thought maybe it was a mod but no. Even after uninstalling and reseting all the settings it still wont work

changing audio format from stereo to automatic fixed the problem for me

**Comment by AdadG** [comment]
> > I have just updated the new version of psc3 and it completely broke the audio on p5 It sounds distorted, I thought maybe it was a mod but no. Even after uninstalling and reseting all the settings it still wont work
> 
> changing audio format from stereo to automatic fixed the problem for me

Thank you.

**Comment by ViviNyan** [comment]
> 

Having most likely the same issue here but with Drakengard 3, game worked fine before update, doing this did not help

**Comment by MLGHat** [comment]
Didn't fix for me. After the update my audio in Demon's Souls is completely broken. I've tried every option for audio settings.
Is there any way I can reverse the update?

**Comment by Megamouse** [comment]
stop posting on this PR

**Pull request merged**